packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -40,6 +40,12 @@ const simpleData = [
         lastName: 'Simpson',
         status: 'success',
         isChild: true
+    },
+    {
+        firstName: 'Abbey',
```

Comment:
I'd never heard of a Simpsons character named Abbey. In the process of searching for a [long-lost Simpsons relative](https://simpsons.fandom.com/wiki/Category:Long-lost_relatives) to replace this with, I learned that Homer [has a half-sister named Abbey](https://simpsons.fandom.com/wiki/Abbey). Well played! 👍

Reply:
That was all @rajsite. He made that suggestion in a previous comment on this PR -- I personally know absolutely nothing about the Simpsons 😄